src/pet.js

```diff
@@ -0,0 +1,88 @@
+const INCREASE_AGE = 1;
+const MAXIMUM_AGE = 30;
+
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0;
+const INCREASE_FITNESS = 4;
+const DECREASE_FITNESS = 3;
+const CHECKUP_FITNESS = 3;
+
+const MINIMUM_HUNGER = 0;
+const MAXIMUM_HUNGER = 10;
+const DECREASE_HUNGER = 3;
+const INCREASE_HUNGER = 5;
+const CHECKUP_HUNGER = 5;
+
+const ERROR_MESSAGE = 'Your pet is no longer alive :('
+
+function Pet(name) {
```

Comment:
This is a general point and not specific to this line, but its very unusual to have 3 spaces of indentation. The convention is either 2 or 4 (by far the most common Ive seen is 2 but theres no definitive rule about this, it tends to vary by organisation). You should update this to use one of these values, and make sure its the same across the whole project for consistency